### PR TITLE
Add CONTRIBUTING.md

### LINKED ISSUE #104: Add a CONTRIBUTING.md File for the repository
Adding a contributing guide will help contributors follow best practices.
I would like to work with maintainers help them adding adding a CONTRIBUTING.md file for the repository.

### PR DESCRIPTION
**Pull request by utkarshshendge**
#### This is the first iteration of CONTRIBUTING.md for [talawa-api](https://github.com/PalisadoesFoundation/talawa-api).
Since many new contributors including myself are not aware with the general workflow of The Palisadoes Foundation, this CONTRIBUTING.md depends on [PR](https://github.com/PalisadoesFoundation/talawa/pull/97) by @jordanliu.
fixes #104

### CODE REVIEW

**Comment by utkarshshendge** [comment]
Hey @heyjordn,  please review this PR.
Thanks!!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+* Hotfix branches (hotfix/branch-name)
+
+Detailed document containing how GitFlow works: https://nvie.com/posts/a-successful-git-branching-model/
+
+### Contributing Code
+Code contributions to Talawa come in the form of pull requests. These are done by forking the repo and making changes locally. 
+
+The process of proposing a change to Talawa can be summarized as:
+1. Fork the Talawa API repository and branch off `master`
+1. Make the desired changes to the Talawa API source
```
Comment (Sagar2366) [inline CONTRIBUTING.md:48]: Please add numbering to this points correctly.

**Comment by Sagar2366** [comment]
Can you combine all these instructions in existing contribution.md file ? Content LGTM.

**Comment by utkarshshendge** [comment]
Hi, @Sagar2366 you are speaking about which CONTRIBUTING.md file? 
Talawa-api repo does not have a CONTRIBUTING.md  file.

**Comment by Sagar2366** [comment]
I was referring to contribution.md file here - https://github.com/PalisadoesFoundation/talawa-api/blob/master/configuration.md

**Comment by utkarshshendge** [comment]
> I was referring to contribution.md file here - https://github.com/PalisadoesFoundation/talawa-api/blob/master/configuration.md

@Sagar2366 , you are referring to [configuration.md]( https://github.com/PalisadoesFoundation/talawa-api/blob/master/configuration.md) file.
If you think its necessary to add link of configuration.md file in CONTRIBUTION.md then I will add it .

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* Release branches (release/1.XX)
+* Bug branches  (bugfix/branch-name)
+* Hotfix branches (hotfix/branch-name)
+
+Detailed document containing how GitFlow works: https://nvie.com/posts/a-successful-git-branching-model/
+
+### Contributing Code
+Code contributions to Talawa come in the form of pull requests. These are done by forking the repo and making changes locally. 
+
+The process of proposing a change to Talawa can be summarized as:
```
Comment (Sagar2366) [inline CONTRIBUTING.md:46]: LGTM. I would like to see more detailed instructions(commands to use) here to follow by new contributors.

Comment (utkarshshendge) [reply CONTRIBUTING.md:46]: I guess a link to [configuration.md](https://github.com/PalisadoesFoundation/talawa-api/blob/master/configuration.md) would do the work, without increasing the size of CONTRIBUTION.md

**Comment by utkarshshendge** [comment]
Hi @Sagar2366  I have added a link to configuration.md.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Sagar2366) [reply CONTRIBUTING.md:48]: @utkarshshendge, Let me explain it again. I was asking you to add detailed instructions in contributing code section - command to clone repo, add/update new files, commit changes and then push it to remote branch.

**Comment by Sagar2366** [comment]
@utkarshshendge please check my comments in file. Add instructions in contributing code section. For reference visit - https://github.com/PalisadoesFoundation/talawa/blob/master/CONTRIBUTING.md.

**Comment by utkarshshendge** [comment]
Hi @Sagar2366 , I have added the instructions.
Really sorry if I have been causing any trouble.

**Comment by Sagar2366** [comment]
> Hi @Sagar2366 , I have added the instructions.
> Really sorry if I have been causing any trouble.

No problem.

**Pull request merged**